tests/commands/test_init.py

```diff
@@ -114,6 +111,75 @@ def test_all_the_files_kubernetes(tmp_path, config):
 def test_all_the_files_machine(tmp_path, config):
     cmd = InitCommand(config)
     cmd.run(create_namespace(profile="machine"))
+    assert {str(p.relative_to(tmp_path)) for p in tmp_path.glob("**/*")} == {
+        ".gitignore",
+        "charmcraft.yaml",
+        "CONTRIBUTING.md",
+        "LICENSE",
+        "pyproject.toml",
+        "README.md",
+        "requirements.txt",
+        "src",
+        os.path.join("src", "charm.py"),
+        "tests",
+        os.path.join("tests", "integration"),
+        os.path.join("tests", "integration", "test_charm.py"),
+        os.path.join("tests", "unit"),
+        os.path.join("tests", "unit", "test_charm.py"),
+        "tox.ini",
+    }
```

Comment:
Do we have a spot already where we're testing that it gets templated correctly? (Other than the spread tests that use the templates)

If not, could we test for one of the files that the template gets correctly filled?

Reply:
Added.